packages/mux-player-react/package.json

```diff
@@ -28,13 +35,17 @@
     "clean": "shx rm -rf dist/",
     "dev:cjs": "yarn build:cjs --watch=forever",
     "dev:esm": "yarn build:esm --watch=forever",
+    "dev:cjs:lazy": "yarn build:cjs:lazy --watch=forever",
+    "dev:esm:lazy": "yarn build:esm:lazy --watch=forever",
     "dev:types": "yarn build:types -w",
-    "dev": "npm-run-all --parallel dev:types dev:cjs dev:esm",
+    "dev": "npm-run-all --parallel dev:types dev:cjs dev:esm dev:cjs:lazy dev:esm:lazy",
     "build:cjs": "esbuild src/index.tsx --target=es2019 --bundle --sourcemap --metafile=./dist/cjs.json --format=cjs --loader:.css=text --outdir=dist --out-extension:.js=.cjs.js --external:react --external:@mux/* --external:prop-types --define:PLAYER_VERSION=\"'$npm_package_version'\"",
     "build:esm": "esbuild src/index.tsx --target=es2019 --bundle --sourcemap --metafile=./dist/esm.json --format=esm --loader:.css=text --outdir=dist --out-extension:.js=.mjs --external:react --external:@mux/* --external:prop-types --define:PLAYER_VERSION=\"'$npm_package_version'\"",
+    "build:cjs:lazy": "esbuild src/lazy.tsx --target=es2019 --bundle --sourcemap --metafile=./dist/cjs.lazy.json --format=cjs --loader:.css=text --outdir=dist --out-extension:.js=.cjs.js --external:react --external:@mux/* --external:prop-types --define:PLAYER_VERSION=\"'$npm_package_version'\"",
+    "build:esm:lazy": "esbuild src/lazy.tsx --target=es2019 --bundle --sourcemap --metafile=./dist/esm.lazy.json --format=esm --loader:.css=text --outdir=dist --out-extension:.js=.mjs --external:react --external:@mux/* --external:prop-types --define:PLAYER_VERSION=\"'$npm_package_version'\"",
```

Comment:
I KNOW these metafiles are wrong. I just don't know enough about metafiles to do it correctly.

Reply:
Why do you say the metafiles are wrong?

Though, you'd probably want to add the two new metafiles to .npmignore

Reply:
I guess I don't know what metafiles are! And having trouble finding resources on them. I assumed, based on the naming of the other two, that they have some sort of standard naming that I didn't follow here.

But if you don't see any issues, then, um, great 😄

Reply:
The metafiles are json that contain a bunch of information about the bundle. Things like which dependencies are included and what not. It's a file that esbuild generates. You can upload the file to https://www.bundle-buddy.com/esbuild, for example, to see it visually.